hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java

```diff
@@ -424,6 +424,10 @@ public void beforeShutdown(boolean terminate) {
 
     @Override
     public void shutdown() {
+    }
```

Comment:
Do we still need this function? Could just removing EE licence key in `EnterpriseNodeExtension#shutdown()` be an option?

Reply:
yes, we need to shutdown HotRestart and CPPersistense on EE side in this method. I've moved some empty methods to `NodeExtension` interface as default implementation.